src/cfnlint/rules/resources/lmbd/DeprecatedRuntime.py

```diff
@@ -0,0 +1,97 @@
+"""
+  Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this
+  software and associated documentation files (the "Software"), to deal in the Software
+  without restriction, including without limitation the rights to use, copy, modify,
+  merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+  permit persons to whom the Software is furnished to do so.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+  INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+  PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+  HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+from datetime import datetime
+from cfnlint import CloudFormationLintRule
+
+
+class DeprecatedRuntime(CloudFormationLintRule):
+    """Check if EOL Lambda Function Runtimes are used"""
+
+    def __init__(self):
+        """Init"""
+        super(DeprecatedRuntime, self).__init__()
+        self.resource_property_types.append('AWS::Lambda::Function')
+
+    current_date = datetime.today()
+
+    deprecated_runtimes = {
```

Comment:
Maybe move this to a JSON file (or make it part of the Spec files, but that might be interesting for later)?

Reply:
yea, I like that idea but any thoughts about how we would fold this into the spec file?

Reply:
For now just a standalone (like `OnlyOne.json`), to keep it consistent?

Reply:
Moved.